### DISCUSSION TITLE
what software did you use to create these roadmaps?

### DISCUSSION THREAD

**Discussion by fractalboi21** [utilForever/game-developer-roadmap#37] [Q&A] [answered]
what software did you use to create these roadmaps?

**Comment by utilForever** [comment, marked as answer]
I built it using [Balsamiq](https://balsamiq.com/products/mockups/). 😄

**Reply by fractalboi21** [reply]
thanks dude!

**Comment by Sicktek** [comment]
Are they interactive tabs?

**Reply by utilForever** [reply]
Hmm... I don't understand your question. Could you elaborate a bit?

**Reply by breakingtherabbit** [reply]
I think he was asking if your roadmap has interactive tabs like the [Kamran Ahmed's roadmaps](https://github.com/kamranahmedse/developer-roadmap)

**Reply by Sicktek** [reply]
Yes like breakingtherabbit said

**Reply by utilForever** [reply]
It looks great! I'll prepare interactive tabs. Thanks! 🥰

**Reply by CiprianAmza** [reply]
Looks really good. 🗡️

**Reply by breakingtherabbit** [reply]
It was done, @CiprianAmza! [Game Developer (client side)](https://roadmap.sh/game-developer) and [Server Side Game Developer](https://roadmap.sh/server-side-game-developer)
